news/2021-01-14-performance-points-updates.md

```diff
@@ -0,0 +1,127 @@
+
+Today, many scores pull less than 20% of their score from accuracy instead, relying on raw speed as the largest influencing component on how scores are set. This has reduced the meaningful impact of accuracy as a skill considerably, which is a peculiar approach for a rhythm game where accuracy should be king.
+
+As suggested by [**Xexxar**](https://osu.ppy.sh/users/2773526), the following change has been applied:
+
+- A new speed curve has been introduced which factors in Overall Difficulty and accuracy.
+    - New value: ![](/wiki/shared/news/2021-01-14-performance-points-updates/new-speed-curve.png)
+    - Old value: ![](/wiki/shared/news/2021-01-14-performance-points-updates/old-speed-curve.png)
+- The new speed curve, in opposition to the previous one, is no longer linear with respect to accuracy, opting for an exponential instead. This severely nerfs low-accuracy scores, especially around the 60% accuracy mark.
+- Speed value is now scaled with the number of 50s made in a score in order to penalize doubletapping.
+    - If there is less than one 50 per 500 objects, the factor is a constant 1 (the speed value is unchanged; technically it's 0.98 to the zeroth power - which is 1)
+    - If there is more than one 50 per 500 objects, the factor is 0.98 to the power of (50 count - total count / 500).
+    - Old value: N/A
+
+To help illustrate these changes, consult the graph images below:
+
+Speed curve featuring overall difficulty 8.5 & accuracy on the X axis (red line is the old formula, purple line is the proposed formula, blue line is the difference curve):
+
+![](/wiki/shared/news/2021-01-14-performance-points-updates/accuracy-speed-curve-graph.png)
+
+[View a live version of this graph with an editable OD slider here.](https://www.desmos.com/calculator/8qfjfnto4r)
+
+Speed value scaling with number of 50s (doubletapping penalty, x-axis is the number of 50s, y-axis is the multiplier):
+
+![](/wiki/shared/news/2021-01-14-performance-points-updates/doubletap-penalty.png)
+
+[View a live version of this graph with an editable total hitobject slider here.](https://www.desmos.com/calculator/rzkcvdthtp)
+
+All of this is a lot of math to digest, especially if you don't understand the Minecraft enchanting table language that is math notation. Put shortly, these changes essentially make accuracy more of a big deal at higher levels of play and help prevent the kind of cheese we have seen in recent years with some players submitting low accuracy scores on intense maps and ending up well above their actual skill range because of it.
+
+---
+
+A big thank you to everyone involved in these changes, and especially to [**Xexxar**](https://osu.ppy.sh/users/2773526), whom has floated the major triad of the top-level changes this time around, performing all the formulaic wizardry required to make them tick largely on his own.
```

Comment:
```suggestion
A big thank you to everyone involved in these changes, and especially to [**Xexxar**](https://osu.ppy.sh/users/2773526), who has floated the major triad of the top-level changes this time around, performing all the formulaic wizardry required to make them tick largely on his own.
```
for being an objective pronoun, "whom" does not fit here, since you're referring to the subject